src/utils/products-filter-functions.js

```diff
@@ -8,21 +15,39 @@ const getFilteredByCategory = (products, filterByCategory) => {
   }
 };
 
+/***
+ * This function filters products based on ratings
+ * @params - array of products and array of ratings applied by users
+ * @return - filtered products based on ratings
+ */
+
 const getFilteredByRating = (products, filterByRating) => {
   const minimumRating = Math.min(...filterByRating);
   if (filterByRating.length > 0) {
     return products.filter((product) => product.rating > minimumRating);
   } else {
     return products;
   }
-};;
+};
+
+/***
+ * This function filters products based on pricerange
+ * @params - array of products and price range
+ * @return - filtered products which are greater than the price ranges passed
+ */
 
 const getFilteredByPriceRange = (products, priceRange) => {
   return products.filter(
     ({ price: { discounted: productPrice } }) => productPrice > priceRange
   );
 };
 
+/***
+ * This function filters products based on type of sort passed in param
+ * @params - array of products and sort by price applied on products
+ * @return - sorted products either from low to high or high to low
+ */
+
 const getFilteredByPriceSort = (products, sortBy) => {
   if (sortBy === "highToLow") {
```

Comment:
hightolow can be all caps eg ="HIGH_TO_LOW" just a good practice.

Reply:
Thanks for reviewing. Will change that.